app/views/posts/_expanded.html.erb

```diff
@@ -118,15 +121,27 @@
               <% end %>
             </div>
           <% end %>
-          <% if post.license.present? %>
-            <div class="post--licensing has-margin-left-2">
-              <i class="fas fa-balance-scale" title="License"></i>
-              <%= link_to post.license.name, post.license.url %>
+          <% if post.license.present? or been_edited then %>
```

Comment:
`or` in Ruby is a control flow operator not a logical operator - you want `||`. Also no `then` after an if statement if it's multiple lines (including HTML blocks)